pinax/stripe/actions/invoices.py

```diff
@@ -63,9 +63,116 @@ def pay(invoice, send_receipt=True):
     return False
 
 
+def paid(invoice):
```

Comment:
Would prefer this to be a verb.  Can we change to `mark_paid` so it's clearer what the function is doing?